### ISSUE TITLE
Content Request from Auth-Profile team - Sign-in page for Personal/Contact info

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#34247]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [X] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [ ] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [X] Content support
  - [ ] IA support
  
*Please describe your need and provide any additional documents and/or links that will help us help you. If we need to add a react widget to a page, be sure to provide the code here.*

The Change Address page on VA.gov - https://www.va.gov/change-address/ - will need some minor content updates and a link update to direct users to the new contact page instead of the personal information page.

### 1. Content change needed before sign-in

- [x] Change highlighted "Personal and contact information" to "Contact information"
![image](https://user-images.githubusercontent.com/73354907/146391982-0d62a6c9-94c1-4fc7-9734-43d22664f556.png)

### 2. Content change needed after sign-in

- [ ] Change highlighted "Personal and contact information" to "Contact information"
![image](https://user-images.githubusercontent.com/73354907/146392330-e3fae06e-8d99-42b0-bb19-d263eeb98b64.png)

## Acceptance Criteria
- [ ] Above content changes are completed and deployed to Staging
- [ ] Changes won't  be deployed to Prod until the rest of the Personal Information Revision updates go live sometime in Q1 2022.
---
## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

 
## Supporting Artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/personal-information/personal-information-revision/README.md
- Link to or attach user flows: N/A
- Link to prototype or mockups: https://preview.uxpin.com/82e734535b0ab60a1567348b1234e43b5318ad3a#/pages/144815586/simulate/no-panels?mode=i
- Link to any epics/issues that may be helpful:  https://github.com/department-of-veterans-affairs/va.gov-team/issues/31180

## Will this work be going through the Collaboration Cycle?
- [X] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date:   Feb 2022
- Estimated staging review date:  Late-mid Jan 2022
- Content and IA work needed by:  mid Jan 2022

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name:  Authenticated Experience - Profile
-	OCTO-DE product owner:  Samara Strauss
-	Product manager: Marci McGuire
-	Designer: Christina Gednalske
-	FE engineer: Adam Whitlock
-	Product/team Slack channel: #vsa-authd-exp

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) Slack channel and tag @RLHecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by RLHecht** [comment]
Thanks @MarciMcGuireGCIO. Public Websites will need to change the button link because this is inside a react widget and not in Drupal. Sitewide Content will be able to update the Contact information text though. You'll just need to let us know when this should be published to prod (it won't show in staging because we'll do the work in Drupal).

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @RLHecht! I'll put in a ticket for Sitewide Content as well.

**Comment by Samara-Strauss** [comment]
@MarciMcGuireGCIO does this still need to be open given #34290? I'm genuinely unsure, so let me know if it does. I saw this was checked off for last sprint, so I just want to make sure I understand if there is still work on this specific ticket that needs to be done, or if everything has been moved over to #34290.

**Comment by MarciMcGuireGCIO** [comment]
@Samara-Strauss The button-specific  work was moved to #34290, but the content highlighted in the screen shots still will need to be tweaked. It’s content work, but we’ll eventually have a FE ticket when they tell us what the new text should be.

**Comment by MarciMcGuireGCIO** [comment]
@Samara-Strauss the content request is not done.  I only listed to-dos for myself to create new tickets so I had a place to keep up with requests I needed to create. They did not have estimates for my time and are not listed on the Sprint report.  I am sorry for the confusion.

**Comment by RLHecht** [comment]
@MarciMcGuireGCIO Checking in on this request. We can update this text (not the react widget) when it's ready to go to prod. Public Websites will need to be the ones to update the text in the react widget. Sitewide Content just needs to know when this work should be completed.

**Comment by MarciMcGuireGCIO** [comment]
Hi @RLHecht, thank you! There have been several changes related to the data source for this work, so are not as far along as we anticipated we'd be.   We are shooting for go-live toward the end of March, so it would be great if you can keep this update on your radar.  Thanks! cc @Samara-Strauss

**Comment by RLHecht** [comment]
@Samara-Strauss Should I move this ticket to the icebox for our team?

**Comment by SKasimow** [comment]
Hi @RLHecht - yes thanks. We have a separate ticket for the widget. What sort of lead time do you need to make the change you can control?

**Comment by RLHecht** [comment]
@SKasimow It's a pretty easy lift, so as long as you give us a heads up a few days before we can take care of it. At the top of the ticket it says there are also link updates but I don't see what those are.

**Comment by SKasimow** [comment]
@RLHecht ok thanks. I need to review and update this along with the widget ticket and will tag you when complete.

**Comment by RLHecht** [comment]
@SKasimow Checking in on this and seeing if my team should bring this into the next sprint. Thanks!

**Comment by SKasimow** [comment]
Hi - I just moved out a sprint and will let you know once we start phased roll-out as that should give us a better idea of when we will be at 100% and can make these updates. Thanks for checking!

**Comment by SKasimow** [comment]
Hi Randi - we are planning to go to 100% Mon June 27th; can you make the content changes noted above late Monday or Tuesday?

**Comment by RLHecht** [comment]
No problem! We can let you know when this is live.

**Comment by RLHecht** [comment]
@SKasimow I just want to reconfirm that the only work our team is doing for this is the content update for the before sign-in to change "Personal and contact information" to "Contact information." Thanks! FYSA @aliyahblackmore

**Comment by SKasimow** [comment]
@RLHecht - Yes that is correct Randi! Thank you

**Comment by RLHecht** [comment]
@SKasimow This page is now live with the content change: https://www.va.gov/change-address/ Going to close this ticket.

**Comment by SKasimow** [comment]
Thank you @RLHecht
